shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -146,10 +146,16 @@ FLUTTER_EXPORT
  * This method must only be called after launching the engine via
  * `-runWithEntrypoint:` or `-runWithEntryPoint:libraryURI`.
  *
+ * Although this method returns synchronously, it does not guarantee that a
+ * semantics tree is actually available; it only tells the Flutter framework to
```

Comment:
a semantics tree is available when this method returns. It synchronously ensures that the next frame the Flutter framework creates will have a semantics tree.